### PR TITLE
[judge-d-24] endpoint /contracts renamed to /services

### PR DESCRIPTION
**Pull request by karolkalinski**
I have renamed the endpoint to better reflect our data structure as we have a concept of service in our domain

### CODE REVIEW

File: judge-d-server/src/main/java/dev/hltech/dredd/interfaces/rest/contracts/ServicesController.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import lombok.AllArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+public class ServicesController {
```
Comment (Felipe444) [inline judge-d-server/src/main/java/dev/hltech/dredd/interfaces/rest/contracts/ServicesController.java]: ServiceController

File: judge-d-server/src/main/java/dev/hltech/dredd/interfaces/rest/contracts/ContractsController.java [hunk 2]
```diff
@@ -46,17 +45,7 @@ public ServiceContractsDto create(@PathVariable(name = "provider") String provid
...
-    @ApiResponses(value = {
-        @ApiResponse(code = 200, message = "Success", response = String.class, responseContainer = "list"),
-        @ApiResponse(code = 400, message = "Bad Request"),
-        @ApiResponse(code = 500, message = "Failure")})
-    public List<String> getAvailableServiceNames() {
-        return serviceContractsRepository.getServiceNames();
-    }
-
-    @GetMapping(value = "/contracts/{serviceName}", produces = MediaType.APPLICATION_JSON_VALUE)
+     @GetMapping(value = "/contracts/{serviceName}", produces = MediaType.APPLICATION_JSON_VALUE)
```
Comment (Felipe444) [inline judge-d-server/src/main/java/dev/hltech/dredd/interfaces/rest/contracts/ContractsController.java]: If you moved GET "/services", this should be moved as well.

File: judge-d-server/src/main/java/dev/hltech/dredd/interfaces/rest/contracts/ServicesController.java (hunk 1 above)
Comment (Felipe444) [inline judge-d-server/src/main/java/dev/hltech/dredd/interfaces/rest/contracts/ServicesController.java]: After our discussion I would not move this endpoint to separate controller nor change its path until we have a clear vision what we are trying to achieve.

Comment (karolkalinski) [reply judge-d-server/src/main/java/dev/hltech/dredd/interfaces/rest/contracts/ServicesController.java]: Agreed to keep in line with naming strategy of controllers with plural form.

File: judge-d-server/src/main/java/dev/hltech/dredd/interfaces/rest/contracts/ContractsController.java (hunk 2 above)
Comment (karolkalinski) [reply judge-d-server/src/main/java/dev/hltech/dredd/interfaces/rest/contracts/ContractsController.java]: After our discussion we postpone moving this endpoint to separate controller nor change its path until we have a clear vision what we are trying to achieve.

**Pull request closed without merging**